VeriStand/Data Sharing Custom Device/Source/Help/Build CHM.log

```diff
@@ -0,0 +1,28 @@
+Microsoft HTML Help Compiler 4.74.8702
```

Comment:
I feel like this file should be listed in a `.gitignore` file.

Git supports .gitignore files at any level in the folder structure, so you can put one in `Data Sharing Custom Device/Source/Help` and avoid cluttering the top level one, if you prefer.

Reply:
Done. Deleted file and added a .gitignore for them.